server/logger.js

```diff
@@ -7,12 +7,14 @@ const dateInTimezone = () => {
   return (new Date(Date.now() - offset)).toISOString().substring(0, 10);
 };
 
-winston.add(new winston.transports.Console(nodeEnv === 'test' && { silent: true }));
 if (!['development', 'test'].includes(nodeEnv)) {
   winston.add(new WinstonCloudWatch({
     logGroupName: process.env.DB_NAME, // Group logs by deployment environment
     logStreamName: dateInTimezone, // Group logs by date
+    awsRegion: 'ca-central-1',
```

Comment:
I am not sure if we need this package and transport. 

any logs in lambda goes to cloudwatch anyways. 

something like this should be fine across env - https://github.com/bcgov/enhanced-travel-screening/blob/4765250e38bf55dd175f05e9600800e9b296a236/server/lambda/layer/common/nodejs/custom_modules/logger.js#L5-L13

Reply:
Good idea. I can do that on a separate PR. This fixes the deprecation warning.